nwb_conversion_tools/utils/spike_interface.py

```diff
@@ -1183,10 +1200,18 @@ def write_sorting(
     metadata: dict
         Information for constructing the nwb file (optional).
         Only used if no nwbfile exists at the save_path, and no nwbfile was directly passed.
+    write_as: str (optional, defaults to 'units')
+        How to save the units table in the nwb file. Options:
+        - 'units' will save it to the official NWBFile.Units position; recommended only for the final form of the data.
+        - 'processing' will save it to the processing module to serve as a historical provenance for the official table.
```

Comment:
Oh, I see, you put the docstring here. Could you copy this over?

Reply:
Docstrings added!